plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -203,29 +157,31 @@ class NewCreateServiceModal extends Component {
   }
 
   onMarathonStoreServiceCreateError(errors) {
-    this.setState({errors, isPending: false});
+    this.setState({
+      apiErrors: MarathonErrorUtil.parseErrors(errors),
```

Comment:
The API errors behave differently than validation errors, since they persist between validation and review screens and they get erased in a different cycle. If we mix them we might encounter some problematic situations. For example: we want the API errors to persist when you click "back" to the review screen, but you want to continue to the review screen if there are no validation errors. How would you easily differentiate between them?